---
title: "SAGA (Simplified Association Genomewide Analyses): a user-friendly Pipeline to Democratize Genome-Wide Association Studies"
collection: publications
category: manuscripts
excerpt: "SAGA — a framework that unravels complex multi-omics associations to pinpoint key disease biomarkers."
permalink: /publication/2025-08-29-saga
date: 2025-08-29
venue: 'BioRxiv'
slidesurl: 'http://vivekruhela.github.io/files/saga_slides.pdf'
paperurl: 'http://vivekruhela.github.io/files/saga.pdf'
bibtexurl: 'http://vivekruhela.github.io/files/saga.bib'
citation: 'Cieza, B., Pandey, N., Ruhela, V., Ali, S. and Tosto, G., 2025. SAGA (Simplified Association Genomewide Analyses): a user-friendly Pipeline to Democratize Genome-Wide Association Studies. bioRxiv, pp.2025-08.'
---

Genome-wide association studies (GWAS) have enabled clinicians and researchers to identify genetic variants linked to complex traits and diseases(1). However, conducting GWAS remains technically challenging without bioinformatics expertise due to required data preprocessing, software installation, and analysis scripting (2,3). SAGA is a BASH-based, open-source, fully automated pipeline that integrates three widely adopted tools— PLINK(4), GMMAT(5), and SAIGE(6)—for accessible, robust, and reproducible GWAS. After installation, users only provide standard genotype and phenotype files. The pipeline automates preprocessing, association testing, and visualization, outputting summary statistics, Manhattan plots, and quantile-quantile plot. SAGA enables robust GWAS for users with no scripting experience, expanding access to complex genetic analyses.
